class-udm-updater.php

```diff
@@ -81,7 +83,19 @@ public function __construct($mothership, $muid, $relative_plugin_file, $options
 		add_action('load-plugins.php', array($this, 'load_plugins_php'));
 		add_action('core_upgrade_preamble', array($this, 'core_upgrade_preamble'));
 		
-		add_filter('auto_update_plugin', array($this, 'auto_update_plugin'), 10, 2);
+		/*
+			Maintain compatibility on all versions between WordPress and UDM, specifically since WordPress 5.5 and UDM 1.8.8
+			Due to the new WP's auto-updates interface in WordPress version 5.5, we need to maintain the auto update compatibility on all versions of WordPress and UDM
+		*/
+		if (!$this->get_option('run_replace_auto_update_option_once')) {
```

Comment:
@andtown You'll see that `$this->get_option()` is just a wrapper around the WP options API functions. As such, the option name is in a global namespace. If more than one plugin using this class is installed, they'll clash. Instead, you should fetch the array that's named in `$this->option_name` as elsewhere in the class.

Reply:
@DavidA Ahh yes, you're right. I've moved the `run_replace_auto_update_option_once` into `*_updater_options`.